gateway-service/src/main/java/org/zowe/apiml/gateway/filters/pre/JWTAuthenticationFilter.java

```diff
@@ -0,0 +1,86 @@
+/*
```

Comment:
I think it maybe makes sense to move these two authentication filters into the apiml-security-common library. There are other security related stuff there such as the x509 auth filter.

Reply:
Not sure if it is worth the effort because the filters are using gateway services inside for validation of the authentication source.